english/avatar/air.md

```diff
@@ -11,4 +11,8 @@ The next step is other planets.
 You feel stranded.
 How can I get off of this planet?
 You are desperate and scream, "Oh God!"
+
 Suddenly - [you wake up!](../wake-up/wake-up.md)
+
+Suddenly - [you wake a fire bender!](../fire-bender/fire-bender.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.